js/reference.js

```diff
@@ -4742,6 +4823,14 @@
             return this._contextualize(module._contexts.COMPACT_SELECT_FOREIGN_KEY);
         },
 
+        /**
+         * The _compact/select/foreign_key/bulk_ context of this reference.
+         * @type {ERMrest.Reference}
+         */
+        get compactSelectBulkForeignKey() {
+            return this._contextualize(module._contexts.COMPACT_SELECT_BULK_FOREIGN_KEY);
```

Comment:
I'm unsure how this name should be written. I considered `compactMultiSelectForeignKey` but that was inconsistent with the rest of our "select" contexts which don't specify single or multi.

I think `compactSelectBulkForeignKey` or `compactSelectForeignKeyBulk` both make sense, it comes down to where we want to put the term `bulk`. I could also change this to "multi" instead of bulk but figured bulk makes more sense since this is for "bulk create" compared to how we use "bulk edit" on recordset page.

Reply:
I think what you have is good.